### PR TITLE
Add EtherInc (ETI) Support

### PR DESCRIPTION
**Pull request by suryanshkumr**
# Description
EtherInc is a fork of Ethereum to power the future of blockchain based organizations.

* Homepage: https://einc.io/
* Node: https://api.einc.io/jsonrpc/mainnet
* Explorer: https://explorer.einc.io/
* BIP44 / Chainid 101 ( [satoshilabs/slips#438](https://github.com/satoshilabs/slips/pull/438) )

# Changes
This PR adds EtherInc node support

### CODE REVIEW

**Comment by tayvano** [comment]
See comments left here: https://github.com/MyCryptoHQ/MyCrypto/pull/2246

File: common/features/config/networks/static/reducer.ts [hunk 1]
```diff
@@ -851,7 +852,33 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+    isCustom: false,
+    color: '#3560bf',
+    blockExplorer: makeExplorer({
+      name: 'Etherinc Explorer',
+      origin: 'https://explorer.einc.io'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: ETI_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: As I stated in the previous PR, please include links to the relevant pull request in third party repositories.  It saves time for those reviewing the PR.  Meanwhile, I see your trezor PR was merged three hours ago.  https://github.com/trezor/trezor-common/pull/301  :+1:

File: common/features/config/networks/static/reducer.ts [hunk 2]
```diff
@@ -851,7 +852,33 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+      [SecureWalletName.SAFE_T]: ETI_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: ETI_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: NO PULL REQUEST exists for Ledger.  Please remove LEDGER_NANO_S support from this PR until after you can demonstrate Ledger hardware wallet support.

File: common/features/config/networks/static/reducer.ts [hunk 3]
```diff
@@ -851,7 +852,33 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
     }
   }
-};
+},
+
+  ETI: {
+    id: 'ETI',
+    name: 'Etherinc',
+    unit: 'ETI',
+    chainId: 101,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: I recommend waiting until after WebChain's hardfork before proceeding with this.  It's great that somebody has changed the chain ID, but until the change takes affect, applying this now would be dangerous for users.

Comment (mkrufky) [reply common/features/config/networks/static/reducer.ts]: Also, *very* important:  Please confirm that EtherInc supports `EIP-155`.  This is a requirement for inclusion in MyCrypto.

**Comment by tayvano** [comment]
Webchain fork goes live April 8th so this can go into the April 11th release, so long as the changes requested by @mkrufky are resolved before April 4th.

File: common/features/config/networks/static/reducer.ts (hunk 3 above)
Comment (suryanshkumr) [reply common/features/config/networks/static/reducer.ts]: > Also, _very_ important: Please confirm that EtherInc supports `EIP-155`. This is a requirement for inclusion in MyCrypto.

Yes, EtherInc supports EIP-155

Comment (suryanshkumr) [reply common/features/config/networks/static/reducer.ts]: > I recommend waiting until after WebChain's hardfork before proceeding with this. It's great that somebody has changed the chain ID, but until the change takes affect, applying this now would be dangerous for users.

Ok will wait for webchain fork

File: common/features/config/networks/static/reducer.ts (hunk 2 above)
Comment (suryanshkumr) [reply common/features/config/networks/static/reducer.ts]: i am working on that

**Comment by tayvano** [comment]
Hey, webchain forked so if you want to make the changes requested by mk, we can out this in the next release

File: common/features/config/networks/static/reducer.ts (hunk 2 above)
Comment (suryanshkumr) [reply common/features/config/networks/static/reducer.ts]: i removed LEDGER_NANO_S for now

**Comment by SharonManrique** [comment]
@suryanshkumr Hello! We highly appreciate your contribution and want to get this merged into next release. Is there any update as to when this will be ready for a review by our devs?

**Comment by mkrufky** [comment]
@SharonManrique My concerns here have been addressed.  Unfortunately, however, PR is unmergeable as-is.   It must be rebased against today's master branch.   Please ping me after the rebase and I'll give thumbs.

**Comment by SharonManrique** [comment]
@mkrufky Thank you for the update! Will tag you when this branch has been rebased. :)

**Pull request merged**